terraform/shared/scripts/create_elastic_users.py

```diff
@@ -140,6 +140,17 @@ def store_secret(secret_id, secret_value, role_arn):
             role_arn=role_arn
         )
 
+    # Configure cluster settings
+    cluster_client = elasticsearch.client.ClusterClient(es)
+
+    cluster_client.put_settings(
```

Comment:
Can we use this?
https://registry.terraform.io/providers/elastic/ec/latest/docs/resources/ec_deployment#user_settings_yaml

Reply:
Yes we can. Good spot.